cli.go

```diff
@@ -0,0 +1,290 @@
+/*******************************************************************************
+ * Copyright (c) 2022-2024 Genome Research Ltd.
+ *
+ * Authors:
+ *	- Sendu Bala <sb10@sanger.ac.uk>
+ *	- Michael Grace <mg38@sanger.ac.uk>
```

Comment:
This is a new file, right? Don't think mg38 worked on this.

Reply:
> This is a new file, right? Don't think mg38 worked on this.

It's largely copied from code mg38 was involved with in another repo, so I've left that in.

Reply:
Fair enough.